earth_enterprise/BUILD_Ubuntu.md

```diff
@@ -0,0 +1,26 @@
+# GEE Build Setup for Ubuntu    
+
+## Install Git  
+Install the system default version:
+
+```
+sudo apt-get install git
+```
+_Or_ install the latest version:
+```
+sudo add-apt-repository ppa:git-core/ppa
+sudo apt-get update
+sudo apt-get install git
+```
+
+## Install Git LFS
+Run the following commands: 
+```
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+sudo apt install git-lfs
+```
+
+## Install Required Packages
+```
+sudo apt-get install gcc g++ scons automake autoconf libperl4-corelibs-perl libtool xorg-dev doxygen python-dev alien swig libgtest-dev libstdc++6 libxml2-dev gettext libxinerama-dev libxft-dev libxrandr-dev libxcursor-dev libgdbm-dev libc6 libc6-dev libmng-dev zlib1g-dev libcap-dev libpng12-0 libpng12-dev freeglut3-dev flex libx11-dev bison++ bisonc++ libjpeg-dev libjpeg8-dev python2.7 python2.7-dev python2.7-psycopg2 libogdi3.2-dev libgif-dev libxerces-c-dev libgeos-dev libgeos++-dev libfreetype6 libfreetype6-dev python-imaging libproj-dev python-setuptools libgif-dev libxerces-c-dev libcap-dev libpq-dev openssl libxml2-utils libxmu-dev 
```

Comment:
I just tried this.  On my current Xenial (16.04.3) "python2.7-psycopg2" is now just python-psycopg2 in the ubuntu repo.

Reply:
This has been addressed in OP-515. The library has been renamed in Ubuntu 14.04 also.